### PR TITLE
fix(content): install/build in a copy directory and swap back in

### PR DESCRIPTION
**Pull request by jrgm**
@vladikoff - something like this might be "good enough" to minimize the minutes-long absence of `./dist` during `grunt build`. 

(An alternative approach would be to keep a permanent `/data/fxa-content-server-build`, `rsync` the running copy after the `git pull`, do the build, and then `rsync` the build directory over the running copy and restart).

### CODE REVIEW

**Comment by jrgm** [comment]
I suppose I should note that this probably needs a bit more testing, but feedback on the approach invited.

**Comment by jrgm** [comment]
I built a new box and then put it in a rebuild loop while polling for head.js. As opposed to a minute or two of downtime previously, now there is about  a two second downtime when the content-server is restarted. 

Note: the periodic copy of /data/fxa-content-server to /var/tmp puts a little more pressure on available disk space, but [it's easy to ask for a larger disk](https://github.com/jrgm/fxa-dev/blob/646eef8da3873ab5f628f69c93fad05c29d96dc6/aws/environments/jrgm.yml#L11-L16)

File: aws/environments/jrgm.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+region: us-west-2
+subdomain: jrgm.dev
```
Comment (vladikoff) [inline aws/environments/jrgm.yml]: did you want to commit this file?

File: roles/content/handlers/main.yml [hunk 2]
```diff
@@ -1,14 +1,40 @@
...
   sudo: true
   sudo_user: app
-  command: npm install --production chdir=/data/fxa-content-server
+  command: npm install --production chdir=/var/tmp/fxa-content-server
 
 - name: build fxa-content-server assets
   sudo: true
   sudo_user: app
-  command: grunt build chdir=/data/fxa-content-server
+  command: grunt build chdir=/var/tmp/fxa-content-server
```
Comment (vladikoff) [inline roles/content/handlers/main.yml]: where does the cdn step happen in this case? after all this?

File: aws/environments/jrgm.yml (hunk 1 above)
Comment (jrgm) [inline aws/environments/jrgm.yml]: Yeah, kinda wanted to commit this, but maybe I should drop this since it's not really part of this PR.

File: roles/content/handlers/main.yml (hunk 2 above)
Comment (jrgm) [inline roles/content/handlers/main.yml]: Is the cdn step not part of the `grunt build`?

File: aws/environments/jrgm.yml (hunk 1 above)
Comment (vladikoff) [inline aws/environments/jrgm.yml]: up to you!

File: roles/content/handlers/main.yml (hunk 2 above)
Comment (vladikoff) [inline roles/content/handlers/main.yml]: it will convert the urls as part of grunt build, i was just wondering if the cdn update takes time to process new files in some way. let's have a quick chat in the meeting.

**Comment by jrgm** [comment]
Okay, removed the extra cruft, rebased and squashed. Back to you @vladikoff.

File: roles/content/handlers/main.yml (hunk 2 above)
Comment (shane-tomlinson) [inline roles/content/handlers/main.yml]: > it will convert the urls as part of grunt build, i was just wondering if the cdn update takes time to process new files in some way. let's have a quick chat in the meeting.

If Cloudfront does not have a resource in its cache when requested by the user, the edge server will transparently request the resource from the origin server. No work necessary on our part once the CDN is set up.

Comment (vladikoff) [inline roles/content/handlers/main.yml]: Radical!

**Comment by jrgm** [comment]
@vladikoff, @shane-tomlinson - thumbs up/down? Seems like a good solution to me.

**Comment by vladikoff** [comment]
! Let's try this out!

**Comment by jrgm** [comment]
Since there weren't any pending changes to the fxa-content-server, after letting it run a couple of times, I forced a rebuild on latest.dev.lcip.org (pro tip: delete /data/fxa-content-server/README.md; on the next ansible run, git will detect it is missing, restore it, and notify the handlers to do npm install && grunt build).

Here's what a two-second poll of latest.dev.lcip.org/scripts/head.js shows:
```
13:13:42 HTTP/1.1 200 OK
13:13:44 HTTP/1.1 200 OK
13:13:46 HTTP/1.1 200 OK
13:13:48 HTTP/1.1 502 Bad Gateway
13:13:50 HTTP/1.1 200 OK
13:13:53 HTTP/1.1 200 OK
13:13:55 HTTP/1.1 200 OK
13:13:57 HTTP/1.1 200 OK
13:13:59 HTTP/1.1 200 OK
13:14:01 HTTP/1.1 200 OK
13:14:04 HTTP/1.1 200 OK
```
Not bad, compare to two minutes downtime before.

**Comment by vladikoff** [comment]
> Not bad, compare to two minutes downtime before.

Woohoo!
